src/symbol/cross_tile_symbol_index.js

```diff
@@ -168,6 +181,11 @@ class CrossTileSymbolLayerIndex {
             const zoomIndexes = this.indexes[z];
             for (const tileKey in zoomIndexes) {
                 if (!currentIDs[zoomIndexes[tileKey].bucketInstanceId]) {
+                    for (const key of Object.keys(zoomIndexes[tileKey].indexedSymbolInstances)) {
```

Comment:
is there an advantage to using `for (const key of Object.keys(z))` instead of `for (const key in z))` in this case?

Reply:
I don't think so, it's just left over from an earlier implementation I tried iterating over `Object.entries`. I'll update it to the simpler form.